department-of-veterans-affairs/caseflow
# Move judges index to the users controller

connects #4853 

### Description
Refactor `/queue/judges` to be `/users?role=Judge` so we can easily support future roles and begin to build a user API
Once I merge the PR, we can re-factor FE.


### Testing Plan
1. Go to `/queue/judges`, it should re-direct to `/users?role=Judge`
2. Go to `/users?role=Judge` directly
